src/wp-includes/class-wp-oembed.php

```diff
@@ -539,6 +539,12 @@ public function fetch( $provider, $url, $args = '' ) {
 			if ( is_wp_error( $result ) && 'not-implemented' === $result->get_error_code() ) {
 				continue;
 			}
+			// use youtube-nocookie.com for YouTube embeds
```

Comment:
Comments should be in sentence case.
Start with capital letter.
End with period.